docs/dialect.md

```diff
@@ -173,3 +173,87 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
 
   **Join Capabilities**  <br/>
   Join usage is also defined by capabilities in the manifest file. See the Query section for join relation capabilities [here]({{ site.baseurl }}/docs/capabilities).
+
+### Boolean Support:
+  We can use boolean support functions when a database lacks native boolean support. <br/>
+   **format-true** <br/> 
+  String literal or expression for boolean false value.  
+  ```xml 
+   <format-true value='(1=1)' />
+  ```
+  **format-false** <br/>
+  String literal or expression for boolean false value. 
+  ```xml 
+  <format-false value='(1=0)' />
+  ```
+   **format-bool-as-value** <br/>
```

Comment:
This is only used when CAP_QUERY_BOOLEXPR_TO_INTEXPR is yes

Reply:
Removed this part and kept only the link to the details for capabilities